File: src/main/java/io/github/hdzitao/editstarters/dependency/EditStarters.java
package io.github.hdzitao.editstarters.dependency;


import java.util.Collection;

/**
 * 插件顶层接口
 *
 * @version 3.2.0
 */
public interface EditStarters {

    /**
     * 添加starters
     */
    void addStarters(Collection<Starter> dependencies);

    /**
     * 删除starters
     */
    void removeStarters(Collection<Starter> dependencies);
}

File: src/main/kotlin/io/github/hdzitao/editstarters/springboot/EditStarters.kt
package io.github.hdzitao.editstarters.springboot


/**
 * 插件顶层接口
 *
 * @version 3.2.0
 */
interface EditStarters {
    /**
     * 添加starters
     */
    fun addStarters(dependencies: Collection<Starter>)

    /**
     * 删除starters
     */
    fun removeStarters(dependencies: Collection<Starter>)
}
